config/install.yaml

```diff
@@ -2625,9 +2642,9 @@ data:
     progressive:
       defaultAssessmentSchedule:
         - kind: Pipeline
-          schedule: "10,360,60,10"
+          schedule: "10,200,60,10"
         - kind: MonoVertex
-          schedule: "10,360,60,10"
+          schedule: "10,200,60,10"
```

Comment:
not related to this PR but this will improve the experience of manual testing so that we will fail faster